modules/olm-installing-operator-using-operatorhub.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * operators/olm-adding-operators-to-cluster.adoc
+
+[id='olm-installing-operator-using-operatorhub_{context}']
+= Installing an Operator using the OperatorHub
+
+As a cluster administrator, you can install an Operator using the OperatorHub in
+the {product-title} web console to make it available for all developers using
+your cluster. This procedure uses the etcd Operator as an example.
+
+.Prerequisites
+
+- Access to an {product-title} v4 cluster using an account with `cluster-admin`
+permissions.
+
+.Procedure
+
+. Navigate in the web console to the *Catalog → Operator Hub* page.
+
+. Click *Show Community Operators*.
++
+[NOTE]
+====
+This Operator will be available under the *Red Hat* category in the GA release.
+====
+
+. Choose *etcd* from the list of available Operators, and click *Install*.
+
+. On the *Create Operator Subscription* page, change the *Target* to the
+*global-operators* Operator Group. This makes the Operator available to all
+users and projects that use this {product-title} cluster.
+
+. On the *Catalog → Installed Operators* page, verify that the *etcd*
+ClusterServiceVersion (CSV) eventually shows up and its *Status* ultimately
+resolves to *InstallSucceeded*.
++
+If it does not, switch to the *Catalog → Operator Management* page and inspect
+the *Operator Subscriptions* and *Install Plans* tabs for any failure or errors
+under *Status*. Then, check the logs in any Pods in the *openshift-operators*
```

Comment:
Should "Pods" be lowercase? Elsewhere in this PR it seems to be lower.

Reply:
Fixed the stragglers!